bedrock/settings/base.py

```diff
@@ -128,110 +128,530 @@ def data_path(*args):
 # path to updated p-d data for testing before loading into DB
 PROD_DETAILS_TEST_DIR = str(Path(PROD_DETAILS_JSON_REPO_PATH).joinpath("public", "1.0"))
 
+# Regions defined on the `/locales/` page.
+REGION_AM = "Americas"
+REGION_APAC = "Asia Pacific"
+REGION_EU = "Europe"
+REGION_AF = "Middle East and Africa"
+
+REGIONS = [
+    REGION_AM,
+    REGION_APAC,
+    REGION_EU,
+    REGION_AF,
+]
+
 # Accepted locales
-PROD_LANGUAGES = (
-    "ach",
-    "af",
-    "an",
-    "ar",
-    "ast",
-    "az",
-    "azz",
-    "be",
-    "bg",
-    "bn",
-    "br",
-    "bs",
-    "ca",
-    "cak",
-    "cs",
-    "cy",
-    "da",
-    "de",
-    "dsb",
-    "el",
-    "en-CA",
-    "en-GB",
-    "en-US",
-    "eo",
-    "es-AR",
-    "es-CL",
-    "es-ES",
-    "es-MX",
-    "et",
-    "eu",
-    "fa",
-    "ff",
-    "fi",
-    "fr",
-    "fy-NL",
-    "ga-IE",
-    "gd",
-    "gl",
-    "gn",
-    "gu-IN",
-    "he",
-    "hi-IN",
-    "hr",
-    "hsb",
-    "hu",
-    "hy-AM",
-    "ia",
-    "id",
-    "is",
-    "it",
-    "ja",
-    "ja-JP-mac",
-    "ka",
-    "kab",
-    "kk",
-    "km",
-    "kn",
-    "ko",
-    "lij",
-    "lt",
-    "ltg",
-    "lv",
-    "mk",
-    "ml",
-    "mr",
-    "ms",
-    "my",
-    "nb-NO",
-    "ne-NP",
-    "nl",
-    "nn-NO",
-    "oc",
-    "pa-IN",
-    "pl",
-    "pt-BR",
-    "pt-PT",
-    "rm",
-    "ro",
-    "ru",
-    "sco",
-    "si",
-    "sk",
-    "sl",
-    "son",
-    "sq",
-    "sr",
-    "sv-SE",
-    "ta",
-    "te",
-    "th",
-    "tl",
-    "tr",
-    "trs",
-    "uk",
-    "ur",
-    "uz",
-    "vi",
-    "xh",
-    "zh-CN",
-    "zh-TW",
-    "zu",
-)
+# See: https://en.wikipedia.org/wiki/Wikipedia:WikiProject_Languages/List_of_ISO_639-3_language_codes_(2019)
+LOCALES = {
+    "ach": {
+        "name": "Acoli",
+        "native": "Lwo",
```

Comment:
All of the name data is in the product-details data in the DB. Is there a reason we need to move it into settings?

Reply:
see https://product-details.mozilla.org/1.0/languages.json

Reply:
Oh hey, I didn't know that. I see it now via: `select * from product_details_productdetailsfile where name='languages.json';`. Technically I didn't move it from the db to settings, I moved it from the locales template to settings, filling in the missing ones. But yeah, this is nice and I can adapt this to pull the native names from the db instead. And we don't need the English names but since I was adding the native ones I liked having the English ones as a description. I'll take it back to the drawing board.

Reply:
We have the `product_details` library that makes this easy. You can see it in use here for example:

https://github.com/mozilla/bedrock/blob/aeed6f60b61abde401c8bcd64d024f8e517fe513/bedrock/newsletter/forms.py#L32-L53

Reply:
Also here:

https://github.com/mozilla/bedrock/blob/aeed6f60b61abde401c8bcd64d024f8e517fe513/bedrock/settings/base.py#L367-L380